Formula/m/manticoresearch.rb

```diff
@@ -1,13 +1,51 @@
 class Manticoresearch < Formula
   desc "Open source text search engine"
   homepage "https://manticoresearch.com"
-  url "https://github.com/manticoresoftware/manticoresearch/archive/refs/tags/6.2.12.tar.gz"
-  sha256 "272d9e3cc162b1fe08e98057c9cf6c2f90df0c3819037e0dafa200e5ff71cef9"
-  license "GPL-2.0-only" # License changes in the next release and must be removed from formula_license_mismatches
-  revision 1
+  license all_of: [
+    "GPL-3.0-or-later",
+    "BSD-2-Clause", # xxhash resource
+    "GPL-2.0-only", # wsrep resource
+    { "GPL-2.0-only" => { with: "x11vnc-openssl-exception" } }, # galera resource
+    { any_of: ["Unlicense", "MIT"] }, # unialgo resource
+  ]
   version_scheme 1
   head "https://github.com/manticoresoftware/manticoresearch.git", branch: "master"
 
+  stable do
+    url "https://github.com/manticoresoftware/manticoresearch/archive/refs/tags/6.3.6.tar.gz"
+    sha256 "d0409bde33f4fe89358ad7dbbad775e1499d4e61fed16d4fa84f9b29b89482d2"
+
+    # https://github.com/manticoresoftware/manticoresearch/blob/master/cmake/GetGALERA.cmake#L4-L6
+    resource "galera-dfce7662.zip" do
+      url "https://github.com/manticoresoftware/galera/archive/dfce7662.zip", using: :nounzip
```

Comment:
Need to find way to bypass audit as build script will do MD5 check so cannot use tarball.
```console
❯ brew audit manticoresearch
manticoresearch
  * line 20, col 7: Use GitHub tarballs rather than zipballs (url is https://github.com/manticoresoftware/galera/archive/dfce7662.zip).
  * line 26, col 7: Use GitHub tarballs rather than zipballs (url is https://github.com/percona/wsrep-API/archive/2c211e1.zip).
  * line 40, col 7: Use GitHub tarballs rather than zipballs (url is https://github.com/manticoresoftware/xxHash/archive/71c6c93.zip).
```

Reply:
> Worst case, we just allow build script to continue fetching everything.

Doesn't seem like there is any allowlist yet. Probably will go this route and let build script pull other deps zips and just block the dependencies we use by adding `REQUIRED` to find_package.